src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TagValue.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    /// <summary>
+    /// Defines well known tag values that are used within Azure App Configuration.
+    /// </summary>
+    public class TagValue
```

Comment:
Can you give an example of this usage in the description

Reply:
Perhaps demonstrating the content of a key-value that has a null tag to demonstrate the concept.